src/main/java/core/basesyntax/service/handler/HandlerByActivity.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service.handler;
+
+import java.math.BigDecimal;
+
+public interface HandlerByActivity {
+
+    BigDecimal getSumFruits(BigDecimal bigDecimalActivity);
```

Comment:
```suggestion
public interface OperationHandler {

    void handle(FruitTransaction transaction);
```

Handlers should process transaction and save updated result with dao